Labels issues (#7515)
netdata/netdata: health/REFERENCE.md

However, the issue was not resolved, it's just a matter of the newer data "polluting" the old. For such alarms, it's a
good idea to tell Netdata to not clear the notification, by using the `no-clear-notification` option.

#### Alarm line `label`

Defines the list of labels expected on host, for example, let us suppose that `netdata.conf` is configured with the
following labels:

```yaml
[host labels]
    installed = 20191211
    room = server
```

> but, we also have for the workstations the following `netdata.conf`

Reviewer: ```suggestion
And more labels in `netdata.conf` for workstations:
```

```yaml
[host labels]
    installed = 201705
    room = workstation
```

I have defined inside `netdata.conf` labels to classify hosts and now we can apply alarms for them using labels, for example,

> case I add the following line inside alarms:

Reviewer: Replace this paragraph:

By defining labels inside of `netdata.conf`, you can now apply labels to alarms. For example, you can add the following line to any alarms you'd like to apply to hosts that have the label `room = server`.

```yaml
label: room = server
```

> this will apply alarms only on hosts that have `room = server`. 

Reviewer: Cut this line, since I added the info up above.

It is also possible to combine labels to apply alarm, for example, case I want to raise a specific alarm only for hosts

> inside a room that were installed in a specific time, I can write the following label condition:

Reviewer: You can also combine labels when applying them to alarms. For example, if you want to raise a specific alarm only for hosts inside a room that were installed at a specific time, you can write the following label line:

```yaml
label: room = workstation AND installed = 201705
```

> The `label` is a space-separate list that accepts simple patterns, for example, case I decided to create an alarm 

Reviewer: ```suggestion
The `label` is a space-separated list that accepts simple patterns. For example, you can create an alarm 
```

> that will be applied to all hosts installed in the last decade, I can set label like:

Reviewer: ```suggestion
that will be applied to all hosts installed in the last decade with the following line:
```

    room = workstation
```

Before we proceed, it is necessary to explain something about possible values for labels. The labels do not have restrictions
 for their values, but their name only accepts ASCII characters, but it does not accept all the ASCII values, the 
 label's name only can contain alphabet letters and numbers.

> 

Reviewer 2: i think it was alnum + _ + - ?

Reviewer 3: Thank you, I rewrite the documentation and I moved it for the correct place.
